lua/smart-splits/mux/tmux.lua

```diff
@@ -44,65 +44,27 @@ function M.current_pane_at_edge(direction)
     return false
   end
 
-  local wrap_at_edge = require('smart-splits.config').at_edge == AtEdgeBehavior.wrap
-
   local edge
-  local op
   if direction == Direction.up then
     edge = 'top'
-    op = wrap_at_edge and '<=' or '<'
   elseif direction == Direction.down then
     edge = 'bottom'
-    op = wrap_at_edge and '>' or '>='
   elseif direction == Direction.left then
     edge = 'left'
-    op = wrap_at_edge and '<=' or '<'
   elseif direction == Direction.right then
     edge = 'right'
-    op = wrap_at_edge and '>' or '>='
   else
     return false
   end
 
-  local tmux_expr = string.format('#{pane_id}:#{pane_%s}:#{?pane_active,_active_,_no_}', edge)
-  local panes = tmux_exec({ 'list-panes', '-F', tmux_expr }, true)
-  local active_pane_output_line = vim.tbl_filter(function(line)
-    return not not string.find(line, '_active_')
-  end, panes --[[ @as string[] ]])[1]
-
-  if not active_pane_output_line then
-    -- no active pane?
-    return false
-  end
+  local tmux_expr = string.format('#{&&:#{pane_active},#{pane_at_%s}}', edge)
+  local result = tmux_exec({ 'list-panes', '-f', tmux_expr }, true)
 
-  local active_pane_id = active_pane_output_line:match('(%%[0-9]*):')
-  local active_pane_coord = active_pane_output_line:match(':([0-9]*):')
-  if not active_pane_id or not active_pane_coord then
-    -- no active pane?
-    return false
+  if result and #result == 1 then
+    return true
   end
 
-  local pane_coords = vim.tbl_map(function(line)
-    return line:match(':([0-9]*):')
-  end, panes --[[ @as string[] ]])
-
-  -- sort largest to smallest
-  table.sort(pane_coords, function(a, b)
-    return a > b
-  end)
-
-  local top_coord = pane_coords[1]
-
-  local ok, value = pcall(function()
-    local expr = string.format('return %s %s %s', tonumber(active_pane_coord), op, tonumber(top_coord))
-    return loadstring(expr)()
-  end)
-
-  if not ok then
-    return false
-  else
-    return value
-  end
+  return false
```

Comment:
instead of `if cond then return true else return false end` let's just `return cond`, i.e. `return type(result) == 'table' and #result == 1`